app/src/main/java/io/github/androidacademyglobal/profile/ProfileScreenState.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.androidacademyglobal.profile
+
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Immutable
+
+@Immutable
```

Comment:
This annotation is an optimisation technique as far as I understand

Reply:
yep